app/views/layouts/_header.html.erb

```diff
@@ -15,6 +15,8 @@
   		<ul class="nav navbar-nav">
   			<% if user_signed_in? %>
 
+          <li> <%= link_to "Пригласить", new_invitation_path if current_user.admin? %> </li>
```

Comment:
It think it would look better in some dropdown? In terms of UX/usability, what's the point of always seeing this link?